core/server/master/src/main/java/alluxio/master/metrics/MetricsStore.java

```diff
@@ -215,6 +215,21 @@ public void initCounterKeys() {
       mClusterCounters.putIfAbsent(new ClusterCounterKey(InstanceType.CLUSTER,
           MetricKey.CLUSTER_BYTES_WRITTEN_UFS_ALL.getName()),
           MetricsSystem.counter(MetricKey.CLUSTER_BYTES_WRITTEN_UFS_ALL.getName()));
+
+      MetricsSystem.registerGaugeIfAbsent(
+          MetricsSystem.getMetricName(MetricKey.CLUSTER_CACHE_HIT_RATE.getName()),
+          () -> {
+            long cacheMisses = MetricsSystem.counter(MetricKey.CLUSTER_BYTES_READ_UFS_ALL.getName())
+                .getCount();
+            long total =
```

Comment:
could we do a check for the condition `total == 0`, it would trigger a division by zero error.